pkg/api/apikey.go

```diff
@@ -16,10 +17,15 @@ func GetAPIKeys(c *m.ReqContext) Response {
 
 	result := make([]*m.ApiKeyDTO, len(query.Result))
 	for i, t := range query.Result {
+		expiration := ""
+		if !t.Expires.IsZero() {
+			expiration = t.Expires.Format(time.UnixDate)
```

Comment:
We do not have to format it here as Time is formatted correctly when marshalled into json and you can set the `expiration` type to `*time.Time` so it can be null. If we need specific format of the date for human readability we should do it on the frontend (for example using local time there instead of string that can be in different timezone than the user)

Reply:
I transform it to string because otherwise if `t.Expires` is the zero date and in the frontend the expiration appears to be the January 1, year 1, 00:00:00.000000000 UTC

Reply:
If the `ApiKeyDTO.Expiration` is `*time.Time` instead of `time.Time` you can leave it `nil`. That way it will be sent to frontend empty.